priv/repo/migrations/20170429120741_add_sessions_table.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Hexpm.Repo.Migrations.AddSessionsTable do
+  use Ecto.Migration
+
+  def up do
+    create table(:sessions) do
+      add :token, :binary, null: false
+      add :data, :jsonb, null: false
+
+      timestamps()
+    end
+
+    create index(:sessions, ["(data->>'username')"])
```

Comment:
thoughts about separate column `sessions.username` that we could have a FK constraint to `users.username`? Not sure there's a big advantage, I guess DB design would be a bit cleaner.